### PR TITLE
adjusted minimum transaction amount

### LINKED ISSUE #496: change minimum payment amount to value of a single vote

### PR DESCRIPTION
**Pull request by maikelmclauflin**
fixes #496 
modified logic runtime-wallet to have options and a passed minimum with a default of `1` of whatever unit is being cared about
modified use of runtime-wallet `validateTxSignature` to pass a minimum value

### CODE REVIEW

File: ledger/controllers/wallet.js [hunk 1]
```diff
@@ -254,6 +254,13 @@ v2.read = { handler: (runtime) => { return read(runtime, 2) },
    PUT /v2/wallet/{paymentId}
  */
 
+function voteValueFromSurveyor (surveyor) {
+  const { votes, probi } = surveyor.payload.adFree
+  const bigProbi = new BigNumber(probi)
+  const minimum = bigProbi.dividedBy(votes).dividedBy(1e18).toString()
```
Comment (evq) [inline ledger/controllers/wallet.js]: to be consistent we shouldn't just hardcode 1e18 here and instead should use the runtime currency facilities

Comment (maikelmclauflin) [reply ledger/controllers/wallet.js]: i was thinking about this yesterday and forgot to get to it this morning. thanks!

File: bat-utils/lib/runtime-wallet.test.js [hunk 2]
```diff
@@ -64,6 +64,11 @@ test('validateTxSignature: works', async t => {
   body = { destination: settlementAddress, denomination: { currency: 'BAT', amount: '0.5' } }
   t.throws(() => { wallet.validateTxSignature(info, signTxn(keypair, body)) })
 
+  body = { destination: settlementAddress, denomination: { currency: 'BAT', amount: '0.5' } }
+  wallet.validateTxSignature(info, signTxn(keypair, body), {
+    minimum: 0.1
+  })
+
```
Comment (evq) [inline bat-utils/lib/runtime-wallet.test.js]: can you add a test showing the throw for amount < minimum?

Comment (maikelmclauflin) [reply bat-utils/lib/runtime-wallet.test.js]: for sure!
